src/main/webapp/less/includes/theme.less

```diff
@@ -16,79 +16,1048 @@
+            .flc-pager-pageLink {
+                a {
+                    color: @color1 !important;
+                    text-decoration: none !important;
+                }
+                a.fl-pager-currentPage {
+                    color: @white !important;
+                }
+            }
+        }
+    }
+    .container-list {
+        -moz-transform: translate3d(15px,0,0);
+        -ms-transform: translate3d(15px,0,0);
+        -o-transform: translate3d(15px,0,0);
+        -webkit-transform: translate3d(15px,0,0);
+        background: white;
+        min-height: 100%;
+        position: absolute;
+        top: 50px;
+        transform: translate3d(15px,0,0);
+        width:@width100;
+        z-index: 1;
+        .pagination>li>a {
+            color: @color1 !important;
+        }
+        .pagination>li>span {
+            color: @color1 !important;
+        }
+        .pagination>.active>a {
+            background-color: @color1;
+            border-color: @color1;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+        }
+        .pagination>.active>span {
+            background-color: @color1;
+            border-color: @color1;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+        }
+        .pagination>.flc-pager-pageLink>.fl-pager-currentPage {
+            background-color: @color1 !important;
+            border-color: @color1 !important;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1 !important;
+                border-color: @color1 !important;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1 !important;
+                border-color: @color1 !important;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
```

Comment:
The lines 300-464 look to have well qualified selectors, what's causing the conflict that `!important` is needed?